ruby.rb

```diff
@@ -3,43 +3,68 @@
 
 # 1. Use TWO different Ruby methods to return a new array with all the numbers multiplied by 3. Expected output: [3, 6, 18, 27, 9, 63]
 
-myArray = [1, 2, 6, 9, 3, 21]
+my_array = [1, 2, 6, 9, 3, 21]
 
+#1.
+def times_three array
+  array * 3
+end
+puts times_three my_array
+
+#2.
+puts my_array.map{ |v| v * 3}
 
 
 # 2. Create a method that takes in a sentence and returns a new sentence with the first letter of each word capitalized. Expected output = "Hello There, How Are You?"
 
 sentence = "hello there, how are you?"
 
+def capitalizer input
+  words = input.split(" ")
+  words.map!{|word| word.capitalize}
+  words.join(" ")
+end
 
+puts "This is the result"
+p capitalizer sentence
 
 # 3. Create a method that takes in a string and returns a new string with all the vowels removed. Expected output = " hv n vwls"
 
 no_vowels = "I have no vowels"
 
+letters = no_vowels.chars - ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'] 
+p letters.join
 
 
 # 4. Look at this horrible Ruby code. Fix it to be good Ruby code.
 
-class example
-  constructor(day)
-    @day=day
+class Example
+  def initialize(day)
+      @day=day
+  end
+    
+  def say_hi
+    if(@day=='Friday')
+      puts 'TGIF!'
+    elsif(@day=='Monday')
+      puts 'Its monday again'
+    else
+      puts 'another day'
+    end
   end
-
-  def SayHi
-    if(day === "Friday"){
-      puts "TGIF!"
-    }
-    else if(day === "Monday"){
-      puts "Its monday again"
-    }
-    else{
-      puts "another day"
-    }
 end
 
```

Comment:
Yup, you got it. To follow Ruby styling I would just remove the ( ) after the if/elsif/else statements. Other than that, you got it all.